### ISSUE TITLE
Feature Request: Rollup API Update / Force Start

### ISSUE THREAD

**Issue by alexmaurizio** [elastic/elasticsearch#45031]
**Describe the feature**: This request for the Rollup API is actually two different requests:

1) **Allow rollup job to immediately start indexing phase, without waiting for the cron**
I encountered problems when experimenting with a 400mil document cluster in production (Elastic Cloud) where I tested some rollup jobs. Experimenting with them usually relies on letting them start during night and checking the results the day after. Since I am still balancing aggregation time vs granularity, I need to run some different rollup jobs on the data. 
Not having the possibility to immediately start the job is extremely limiting, but I suppose it should not be that difficult to launch the indexing state on user request, instead of waiting up the cron.

2) **Allow rollup job updates limited to some configuration changes**
I understand that having different configuration pattern on rollups may lead to corrupt data, misleading config and so on, and that's why the restriction for the rollup data ID in the index is in place: to forbid having different jobs running with the same ID in different times and messing up the data.
But there are some config settings that are not affecting the index process at all, like cron timing, page size and timeout configuration: those are actually impossible to change on an already-made rollup job, thus de-facto breaking tweaking capabilities on the job (I would like to change pagination size to better suit my cluster or hour of running and I'm not allowed to do so without re-creating the whole job, which at that point should have a different ID and thus should start rolling back the data from the beginning, which can take days).

TL;DR Feature requests
1) Add a **POST _rollup/job/{JOB-ID}/_forcestart** - or similar - API that allows the user to put the requested rollup job immediately in indexing state, without waiting for the cron
2) Add a **PUT _rollup/job/{JOB-ID}/_settings** - or similar - API that allows user to set up non-vital, non-indexing related configurations for the related job, like page size, timeout and cron setting

Thanks!

**Comment by polyfractal** [comment]
Thanks for the ticket @alexmaurizio!  ++ to both of these.

Adding a "force start" API, or a flag on creation, makes a lot of sense to me.  I don't think it would be very difficult either, we just need to trigger the indexer right away instead of waiting for the cron.  And since only one indexer can be running, even if the cron triggers immediately afterwards nothing bad would happen.

Limited modifications of the config is definitely something we want to support, and a pretty high priority item.  See the third bullet point here: https://github.com/elastic/elasticsearch/issues/42720

**Comment by alexmaurizio** [comment]
Thanks for the feedback!
Didn't know the config modification was already proposed, looking forward to see those features implemented. Would love to help, if only I had some spare time.. might probably check later if there is something a "beginner" (on elastic core) could do.

As for the force start, my thought exactly was for the indexing force start won't cause problems with the auto cron start since I noticed myself when the cron re-triggered and the job was already indexing, nothing bad would happen. So that was my assumption for the "it should be easy enough" phrase.

I'll keep an eye on the issue. Thanks again!

**Comment by csoulios** [comment]
This feature request is about the current experimental version of rollups. The team has decided to implement this functionality from scratch as the downsample operation of the TSDB project (https://github.com/elastic/elasticsearch/issues/74660). 

As we focus all our efforts into implementing downsampling for TSDB, the team has decided not to proceed with adding new functionality to the current experimental rollups. 

Therefore, I am closing this issue.
